src/tox_lsr/test_scripts/runcontainer.sh

```diff
@@ -124,25 +124,28 @@ setup_plugins() {
     if [ -z "${ANSIBLE_CONNECTION_PLUGINS:-}" ] || [ ! -f "$ANSIBLE_CONNECTION_PLUGINS/podman.py" ]; then
```

Comment:
do we need to change this condition?  or does the presence of `podman.py` imply the presence of `buildah.py`?

Reply:
Thanks for spotting -- it happens to work, but looks weird and is accidental. Fixed.